contrib/experimental/great_expectations_experimental/expectations/expect_column_values_to_be_present_in_other_table.py

```diff
@@ -0,0 +1,227 @@
+     the values of CUSTOMER_ID in order_table_1 are present in the CUSTOMER_ID column of customer_table.
+
+    "order_table_1": {
+         "ORDER_ID": ["aaa", "bbb", "ccc"],
+         "CUSTOMER_ID": [1, 1, 3],
+     }
+     "customer_table": {
+         "CUSTOMER_ID": [1, 2, 3],
+
+     }
+
+     However, in the second example, referential integrity is not preserved, because there are two values (4 and 5) in the CUSTOMER_ID
+     column of order_table_2 that are not present in the CUSTOMER_ID column of customer_table.
+
+     "order_table_2": {
+         "ORDER_ID": ["ddd", "eee", "fff"],
+         "CUSTOMER_ID": [1, 4, 5],
+     }
+     "customer_table": {
+         "CUSTOMER_ID": [1, 2, 3],
+
+     }
+     ExpectColumnValuesToBePresentInAnotherTable will PASS for example 1 and FAIL for example 2.
+
+     Args:
+         template_dict: dict containing the following keys:
+             foreign_key_column: foreign key-column of current table that we want to validate.
+             foreign_table: foreign table.
+             primary_key_column_in_foreign_table: key column for primary key in foreign table.
+    """
+
+    library_metadata = {
+        "maturity": "experimental",
+        "tags": ["table expectation", "multi-table expectation", "query-based"],
+        "contributors": [
+            "@great_expectations",
+        ],
+        "requirements": [
+            "sqlalchemy",
+            "snowflake-sqlalchemy",
+            "snowflake-connector-python",
+        ],
+        "has_full_test_suite": False,
+        "manually_reviewed_code": True,
+    }
+
+    metric_dependencies = ("query.template_values",)
+    template_dict: dict
+    query = """
+        SELECT count(1) FROM (
+        SELECT a.{foreign_key_column}
+        FROM {active_batch} a
+        LEFT JOIN {foreign_table} b
+            ON a.{foreign_key_column} = b.{primary_key_column_in_foreign_table}
+        WHERE b.{primary_key_column_in_foreign_table} is NULL)
+        """
+    success_keys = (
+        "template_dict",
+        "query",
+    )
+    domain_keys = ("query", "batch_id", "row_condition", "condition_parser")
+
+    default_kwarg_values = {
+        "catch_exceptions": False,
+        "meta": None,
+        "query": query,
+    }
+
+    @classmethod
+    @override
+    @renderer(renderer_type="renderer.prescriptive")
+    def _prescriptive_renderer(
+        cls,
+        configuration: Optional[ExpectationConfiguration] = None,
+        result: Optional[ExpectationValidationResult] = None,
+        runtime_configuration: Optional[dict] = None,
+    ) -> List[RenderedStringTemplateContent]:
+        runtime_configuration = runtime_configuration or {}
+        styling = runtime_configuration.get("styling")
+
+        template_dict = configuration.kwargs.get("template_dict")
+
+        template_str = "No values in the column $foreign_key_column are expected to be missing in table $foreign_table 's $primary_key_column_in_foreign_table column."
```

Comment:
What do you think about flipping so that it is asserting a positive? Something like:
```suggestion
        template_str = "$foreign_key_column values exist in table $foreign_table 's $primary_key_column_in_foreign_table column."
```
Also, what do you think about using the column first? Looks like it might be more consistent with some of the other core expectations, here is a sample:
![image](https://github.com/great-expectations/great_expectations/assets/9903066/2870aefa-7b40-4187-ad65-2f1ec6c5fc5d)

Reply:
I tried to display the column value, but unfortunately the `QueryExpectation` is a category of its own, so I think it gets grouped with the Table Expectations